agent/testflinger_agent/agent.py

```diff
@@ -32,13 +32,19 @@ def __init__(self, client):
     def _post_initial_agent_data(self):
         """Post the initial agent data to the server once on agent startup"""
 
-        location = self.client.config.get("location", "")
         self._post_advertised_queues()
         self._post_advertised_images()
 
-        queues = self.client.config.get("job_queues", [])
         identifier = self.client.config.get("identifier")
-        agent_data = {"queues": queues, "location": location}
+        location = self.client.config.get("location", "")
+        provision_type = self.client.config.get("provision_type")
```

Comment:
is it guaranteed to have a value or should we have some default?

Reply:
Good idea, fixed. thanks!